documentation/examples/prometheus-mesos-cluster.yml

```diff
@@ -0,0 +1,50 @@
+global:
+  scrape_interval:     15s # By default, scrape targets every 15 seconds.
+
+  # Attach these labels to any time series or alerts when communicating with
+  # external systems (federation, remote storage, Alertmanager).
+  external_labels:
+    monitor: 'mesos_cluster'
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  - job_name: 'mesos_nodes'
+    # If you have mesos dns configured on the machine running 
+    # prometheus you can use dns to find all nodes to pull from
+    dns_sd_configs:
+    - names:
+      - slave.mesos
+      type: 'A'
+      # Port to pull from
+      # Default port for node exporter
+      port: 9100
+  
+  # Find the master nodes
+  - job_name: 'mesos_masters'
+    dns_sd_configs:
+    - names:
+      - master.mesos
+      type: 'A'
+      port: 9100
+
+  # Pull in apps from marathon
+  - job_name: 'marathon_apps'
+    marathon_sd_configs:
+    - servers:
+      # Mesos dns will resolve the marathon master
+      # 8080 is the default port for marathon
+      - 'http://marathon.mesos:8080'
+
+    relabel_configs:
+    # Only scrape apps that have a label of 'MONITOR' set to 'prometheus'
+    # in the marathon app definition
+    - source_labels: [__meta_marathon_app_label_MONITOR]
+      action: keep
+      regex: prometheus
+    # Add marathon_app label
+    - source_labels: [__meta_marathon_app]
+      target_label: marathon_app
+    # Add a label that detects docker image
```

Comment:
This shouldn't be a target label, as you already have the app as job and this doesn't add distinctness on top of that.